plugins/inputs/mem/memory.go

```diff
@@ -61,6 +62,11 @@ func (s *MemStats) Gather(acc telegraf.Accumulator) error {
 		"write_back":        vm.Writeback,
 		"write_back_tmp":    vm.WritebackTmp,
 	}
+
+	if runtime.GOOS == "freebsd" {
+		fields["laundry"] = vm.Laundry
```

Comment:
Need to add this to the docs, maybe with a note that it's specific to freebsd

Reply:
We already support active, inactive, and wired which are only valid on OS X and BSD.  They're not handled conditionally at runtime.  Maybe this doesn't need to be either.

Reply:
Active/inactive are set on Linux, despite the comment in gopsutil.  We really shouldn't be adding fields that aren't set, so I went through the stats again in more detail.  One situation I'm not handling yet is the non-cgo version of Darwin.